Valour/Server/Database/Items/Planets/PlanetInvite.cs

```diff
@@ -91,7 +91,7 @@ public static async Task<IResult> GetRouteAsync(HttpContext ctx, [FromRoute] str
 
         PlanetHub.NotifyPlanetItemChange(invite);
 
-        return Results.NoContent();
+        return Results.Text(invite.Code);

```

Comment:
If you want to return something, fine - But return the uri and entire created object like in other places with Results.Created